rcl_interfaces/msg/FloatingPointRange.msg

```diff
@@ -1,14 +1,13 @@
 # Represents bounds and a step value for a floating point typed parameter.
 
 # Lower bound for valid values, inclusive.
```

Comment:
We should probably change this to `Left bound` and `Right bound` below instead of `Upper bound`.

Reply:
I agree that Lower/Upper is not the right choice. Using left/right and lower/upper makes me think in minimums and maximums. 
My proposal would be to keep with the from/to idea and say: 

'Start value for valid values range' and 'End value for valid values range'

Reply:
Sounds good to me.